pkg/dm-crypt/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM linuxkit/alpine:3683c9a66cd4da40bd7d6c7da599b2dcd738b559 AS mirror
+RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
+RUN apk add --no-cache --initdb -p /out  \
+    alpine-baselayout \
+    cryptsetup \
+    e2fsprogs
+
+
+# Remove apk residuals
+RUN rm -rf /out/etc/apk /out/lib/apk /out/var/cache
+
+FROM scratch
+ENTRYPOINT []
+WORKDIR /
+COPY --from=mirror /out/ /
+
+COPY crypto.sh /crypto
```

Comment:
@rn that was my request. This is in `sh` now, but will likely convert to pure binary (along with everything else we do). Keeping it named just `crypto` made it consistent as "just an executable". Same for `loopy`.

Reply:
Ok, so I'll move it into `/usr/bin` but keep the name as is. I tried to avoid the term `mount` since it's not actually mounting anything - it's mapping a device.